tests/e2e/page-object/home-page.js

```diff
@@ -0,0 +1,46 @@
+exports.HomePage = class HomePage {
+  /**
+   * @param {import('@playwright/test').Page} page
+   */
+  constructor(page) {
+    this.page = page;
+
+    this.loginButton = this.page.locator('#header').getByText('Login');
+    this.registerButton = this.page.locator('#header').getByText('Cadastrar');
+
+    this.recentTab = this.page.locator('a', { hasText: 'Recentes' });
+  }
+
+  async getTitle() {
+    const title = this.page.getByRole('heading', { name: 'Nenhum conteúdo encontrado' });
+    await title.waitFor({ state: 'visible' });
```

Comment:
Pelo o que entendi, isso é um teste para saber se a página foi renderizada, certo?

Se for isso, não faz mais sentido obter um elemento que exista em todas as páginas? Por exemplo, o link `Recentes`. Assim, pode criar uma função já responsável por identificar se a página foi renderizada e usar em todas as páginas.  

Digo isso não apenas para conseguir reutilizar código, mas sim porque esse texto não será encontrado caso exista um conteúdo.